src/MaterialDesignThemes.Wpf/Converters/TopThicknessConverter.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace MaterialDesignThemes.Wpf.Converters;
 
-internal class TopThicknessConverter : IValueConverter
+public class TopThicknessConverter : IValueConverter
```

Comment:
Since this only appears to be referenced by `ObsoleteConverters.xaml` can we leave it as internal as the only usage of it would be from someone needing those obsolete converters?

I would propose, that we also decorate this class with the `[Obsolete]` so it is clear it is no longer used directly by this library.